### ISSUE TITLE
Make 1.0.0 release

### ISSUE THREAD

**Issue by kghbln** [SemanticMediaWiki/SemanticApprovedRevs#3]
Tracking issue for the 1.0.0 release.
- [Release notes](https://github.com/SemanticMediaWiki/SemanticApprovedRevs/blob/master/RELEASE-NOTES.md)

## 
- [x] Finalize RELEASE NOTES
- [x] Update "README.md"
- [x] Update "composer.json"
- [x] Update version number "extension.json"
- [x] Create tag
- [x] Add release to [Wikidata](https://www.wikidata.org/wiki/Q21678422)
- [x] Update [homepage](https://www.mediawiki.org/wiki/Extension:Semantic_Compound_Queries)
- [x] Announce via mail
- [x] Do a release tweet on Twitter
- [x] Run `composer update` on smw.o
- [x] Update "LocalSettings.php"
- [x] Start development of new version 
  - [x] Update "composer.json" (update dev alias)
  - [x] Update "extension.json" (new alpha version)

**Comment by mwjames** [comment]
The release has to happen after SMW 3.1 due to things like https://github.com/SemanticMediaWiki/SemanticMediaWiki/pull/3770.

**Comment by kghbln** [comment]
Thanks for the info.

**Comment by kghbln** [comment]
We need to update CI for the latest AR version, preferably beyond 1.1 due to https://github.com/SemanticMediaWiki/SemanticApprovedRevs/pull/12#issuecomment-516164689

**Comment by krabina** [comment]
AR 1.2 is out...

**Comment by mwjames** [comment]
Unlikely that release is going to happen in 2019 and there is likely
tendency that I will change the way of how the data are stored which
is, instead of polluting an entity with Approv* properties, they are
going to be stored as separate subobject which the user can reach via
`Approved revs.xxx` in a query.

**Comment by hexmode** [comment]
Till this happens, the readme should be updated to indicate that you should use dev-master (or another tag) instead of 1.0.

**Comment by revansx** [comment]
I don't understand how there can be a "dev-master" if it doesn't show up in the list of branches. What am I missing?

**Comment by hexmode** [comment]
See the [documenation](https://getcomposer.org/doc/articles/versions.md#branches):
> If you want Composer to check out a branch instead of a tag, you need to point it to the branch using the special dev-* prefix

**Comment by revansx** [comment]
hmm.. the github repo for SAR only shows 1 branch (master) and there are no tags.. i'm guessing composer pulls code from somewhere else and this repo here on github is partial mirror. or am I really missing something obvious?

**Comment by krabina** [comment]
No, I am quite certain, github is the primary repo for all of SMW extensions.

**Comment by hexmode** [comment]
> hmm.. the github repo for SAR only shows 1 branch (master) and there are no tags.. i'm guessing composer pulls code from somewhere else and this repo here on github is partial mirror. or am I really missing something obvious?

Composer pulls code from the github (or gitlab, or gerrit...) repository.  It retrieves meta-information (what packages are available and from where) from packagist.org.

You can see that the [packagist entry for SAR](https://packagist.org/packages/mediawiki/semantic-approved-revs) does not list any versions except "`dev-master / 0.1.x-dev`" because there are, in fact, no tags on this repository.

The `0.1.x-dev` comes from the composer.json file where it is [specified as an alias](https://github.com/SemanticMediaWiki/SemanticApprovedRevs/blob/9d5ca3041a949d58b892c3a39548c82b428e8af1/composer.json#L32) for `dev-master`.

**Comment by revansx** [comment]
"specified as an alias" <-- thanks!
